client-go/cmd/apps.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/deis/deis/client-go/controller/client"
+	"github.com/deis/deis/client-go/controller/models/apps"
+	"github.com/deis/deis/client-go/controller/models/config"
+)
+
+// AppCreate creates an app with the application
+func AppCreate(id string, buildpack string, remote string, noRemote bool) error {
+	c, err := client.Load()
+
+	fmt.Print("Creating Application... ")
+	quit := progress()
+	app, err := apps.Create(*c, id)
+
+	quit <- true
+	<-quit
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("done, created %s\n", app.ID)
+
+	if buildpack != "" {
+		configValues := map[string]string{
+			"BUILDPACK_URL": buildpack,
+		}
+		err = config.Set(*c, app.ID, configValues)
+
+		if err != nil {
+			return err
+		}
+	}
+
+	if !noRemote {
+		return c.CreateRemote(remote, app.ID)
+	}
+
+	fmt.Println("remote available at", c.RemoteURL(app.ID))
+
+	return nil
+}
+
+// AppsList lists apps on the controller
+func AppsList() error {
+	c, err := client.Load()
+
+	if err != nil {
+		return err
+	}
+
+	apps, err := apps.List(*c)
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("=== Apps")
+
+	for _, app := range apps {
+		fmt.Println(app.ID)
+	}
+	return nil
+}
+
+// AppInfo prints info about app
+func AppInfo(appID string) error {
+	c, err := client.Load()
+
+	if err != nil {
+		return err
+	}
+
+	if appID == "" {
+		appID, err = c.DetectApp()
+
+		if err != nil {
+			return err
+		}
+	}
+
+	app, err := apps.Get(*c, appID)
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("=== %s Application\n", app.ID)
+	fmt.Println("updated: ", app.Updated)
+	fmt.Println("uuid:    ", app.UUID)
+	fmt.Println("created: ", app.Created)
+	fmt.Println("url:     ", app.URL)
+	fmt.Println("owner:   ", app.Owner)
+	fmt.Println("id:      ", app.ID)
+
+	return nil
+}
+
+// AppOpen opens an app in the default webbrowser
+func AppOpen(appID string) error {
+	c, err := client.Load()
+
+	if err != nil {
+		return err
+	}
+
+	if appID == "" {
+		appID, err = c.DetectApp()
+
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
This stanza starting with `c, err := client.Load()` is repeated in several functions. Can we make it a helper function to keep the code DRY?